### ISSUE TITLE
Error occurred when executing SAM Image Mask:  'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by flyricci** [frankchieng/ComfyUI_MagicClothing#62]
inpaint's workflow does not change any parameters, and the following error is reported

File "/root/ComfyUI/execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "/root/ComfyUI/execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "/root/ComfyUI/execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "/root/ComfyUI/custom_nodes/was-node-suite-comfyui/WAS_Node_Suite.py", line 11469, in sam_image_mask
masks, scores, logits = predictor.predict(
File "/root/ComfyUI/venv/lib/python3.10/site-packages/segment_anything/predictor.py", line 154, in predict
masks, iou_predictions, low_res_masks = self.predict_torch(
File "/root/ComfyUI/venv/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
return func(*args, **kwargs)
File "/root/ComfyUI/venv/lib/python3.10/site-packages/segment_anything/predictor.py", line 229, in predict_torch
low_res_masks, iou_predictions = self.model.mask_decoder(
File "/root/ComfyUI/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
return self._call_impl(*args, **kwargs)
File "/root/ComfyUI/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
return forward_call(*args, **kwargs)
File "/root/ComfyUI/custom_nodes/comfyui_segment_anything/sam_hq/modeling/mask_decoder_hq.py", line 124, in forward
vit_features = interm_embeddings[0].permute(0, 3, 1, 2) # early-layer ViT feature, after 1st global attention block in ViT

**Comment by il-vanzo** [comment]
How did you solve this?
